### ISSUE TITLE
old_release returns CentOS-Stream

### ISSUE THREAD

**Issue by jtome7** [oracle/centos2ol#139]
Can I edit the centos2ol.sh script in a few places where it checks for "old_release" in order to run it with CentOS-Stream-8 installed?

# rpm -q --whatprovides /etc/redhat-release
centos-stream-release-8.6-1.el8.noarch

REPOS include lots of CentOS-Stream******.repo

**Comment by Djelibeybi** [comment]
Not really. CentOS Stream has diverged so you're likely to run into cascading problems as time goes on.

**Comment by scoter-oracle** [comment]
Script has been revamped. Please re-test and re-open the issue if it still fails.
